validator/validator_test.go

```diff
@@ -501,6 +517,7 @@ testdata/invalid_validation_localized_message.proto:54:26: message must always r
 			}
 
 			if diff := cmp.Diff("\n"+got, test.expected); diff != "" {
+				fmt.Println(got)
```

Comment:
Is this intentional, or is it just a leftover from debugging? 🤔

Reply:
I forgot to remove it !! Thanks 🙏